elements/lisk-p2p/src/peer_selection.ts

```diff
@@ -63,8 +63,23 @@ export const selectPeersForRequest = (
 
 export const selectPeersForSend = (
 	input: P2PPeerSelectionForSendInput,
-): ReadonlyArray<P2PDiscoveredPeerInfo> =>
-	shuffle(input.peers).slice(0, input.peerLimit);
+): ReadonlyArray<P2PDiscoveredPeerInfo> => {
+	const shuffledPeers = shuffle(input.peers);
+	// tslint:disable: no-magic-numbers
+	const halfPeerLimit = Math.round((input.peerLimit as number) / 2);
+
+	// TODO: Get outbound string from constants.ts file.
+	const selectedOutboundPeers = shuffledPeers
+		.filter((peerInfo: P2PDiscoveredPeerInfo) => peerInfo.kind === 'outbound')
```

Comment:
I recommend to follow the pattern we introduced in `getPeers()` from PeerPool. We use `typeOf` and `instanceOf`. That way we don't have to create _kind_ constants.

Reply:
The object provided here is of type `P2PPeerInfo/P2PDiscoveredPeerInfo` so we can't use `instanceof`. Another reason for this which we discussed in the meeting is that in the near future we could also add this `kind` property to each P2PPeerInfo object which is returned from the `p2p.getConnectedPeers()` method.